modules/statistics/statistics.js

```diff
@@ -211,6 +218,23 @@ Statistics.prototype.startCallStats = function (session, settings) {
 };
 
 /**
+ * Removes the callstats.io instances.
+ */
+Statistics.prototype.stopCallStats = function () {
+    if(this.callStatsIntegrationEnabled && this.callstats) {
```

Comment:
The condition this.callStatsIntegrationEnabled appears superfluous to me, this.callstats looks like it's enough.